js/doc.js

```diff
@@ -113,13 +114,18 @@ function SchemaPart({ key, property, parent, parentSlug }) {
         type = `[]object`;
       }
     }
+    if (property.Enum) {
+      type = type + ': enum';
+      enumvals = property.Enum;
```

Comment:
Returning the enum variants or an empty array:

```suggestion
      enumvals = property.Enum ?? [];
```

Reply:
We can use this in line 166.

Reply:
Ahh, now I get it